matrix.py

```diff
@@ -0,0 +1,47 @@
+from typing import Self
+
+
+class Matrix:
+    """
+    Matrix class.
+
+    Represents the mathematical matrix and implements the method of the sum of two matrix.
+
+    """
+
+    def __init__(self: Self, init_data: list[list[int | float]] | tuple[list[int | float], ...]):
```

Comment:
```suggestion
    def __init__(self: Self, init_data: Sequence[Sequence[int | float]]:
```
You can use `Sequence` from typing

Reply:
Done